### PR TITLE
R-igraph: update to 1.4.0

### PR DESCRIPTION
**Pull request by szhorvat**
#### Description

 - Update R-igraph to 1.4.0
 - Remove unneeded `port:pkgconfig` dependency from `R-pkgconfig` (unrelated to `pkgconfig`) and from `R-igraph`

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on
macOS 10.14.6 18G9323 x86_64
Xcode 11.3.1 11C504

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [ ] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

**Comment by barracuda156** [comment]
@szhorvat Great.

BTW, once this is merged, we can add support for tests: https://github.com/macports/macports-ports/pull/17660
For `igraph` a couple of new packages will be needed though.

**Review by barracuda156 — APPROVED** [review]

File: R/R-pkgconfig/Portfile [hunk 1]
```diff
@@ -14,5 +14,3 @@ checksums           rmd160  c6360d0c6237ab56c553b3a515ca9bf452557e43 \
                     sha256  c371a82737112adcae905ddec116d1e5f8c779fb802e681cfe1f8c7027bfeacf \
                     size    6482
 supported_archs     noarch
-
-depends_lib-append  port:pkgconfig
```
Comment (harens) [inline R/R-pkgconfig/Portfile:18]: Modifying library dependencies generally warrants a revbump ([see docs](https://guide.macports.org/#reference.keywords)).

Comment (barracuda156) [reply R/R-pkgconfig/Portfile:18]: @harens Thank you!

Comment (szhorvat) [reply R/R-pkgconfig/Portfile:18]: Thanks for point this out. It is now fixed.

**Review by harens — APPROVED** [review]
Thanks @szhorvat for updating this! 👍

**Comment by barracuda156** [comment]
Well, I should have actually tried building it: https://github.com/igraph/rigraph/issues/669
My bad.

**Comment by szhorvat** [comment]
I did build it, and it built fine. Let's continue in the issue you opened.

**Pull request merged**